### DISCUSSION TITLE
Which famous playwright wrote the tragedy "Romeo and Juliet," a timeless tale of young love and family feuds ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#58] [Galaxy Qs] [answered]
Which famous playwright wrote the tragedy "Romeo and Juliet," a timeless tale of young love and family feuds ?

**Comment by opticaldrive** [comment, marked as answer]
William Shakespeare?

**Reply by Sriansh-raj** [reply]
Congratulations @V205Arduino you just got the `Galaxy Brain🥉` achievement. 🎉

Don't forget to create a PR and update your name in [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) to create a history 😃

**Reply by opticaldrive** [reply]
Will do!
